### DISCUSSION TITLE
推车的轮子无法合成 Can't craft wheel

### DISCUSSION THREAD

**Discussion by forgeters** [Rad586/Forget-Me-Not#73] [Q&A] [answered]
按照模组给出的合成表无法合成
![image](https://github.com/Rad586/Forget-Me-Not/assets/98031979/3a2a4d14-7013-4ec2-8ab8-501615f72f7f)
是否移除了相应的合成表，那轮子要如何获得呢

**Comment by Rad586** [comment, marked as answer]
搜索并删除`wheel.json`，我不知道为什么把这个合成表去掉了
Search and delete `wheel.json`
